extension/lib/shopify/AdminClient.js

```diff
@@ -0,0 +1,56 @@
+const ShopifyClient = require('shopify-api-node')
+const STOREFRONT_ACCESS_TOKEN_TITLE = 'shopgate-app-access-token'
+
+class ShopifyAdminClient extends ShopifyClient {
+  /**
+   * @param {string} adminAccessToken
+   * @param {string} shopName
+   */
+  constructor (adminAccessToken, shopName) {
+    super({shopName: shopName, accessToken: adminAccessToken})
+  }
+
+  /**
+   * @param {string} id
+   * @return {Promise.<void>}
```

Comment:
please replace all "Promise.<" with "Promise<"

Reply:
It can also be changed to "number" instead of void

Reply:
done